core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -167,7 +171,7 @@ public int run(final String[] args,
             final HashMap<Object, Object> consumerConfig = new HashMap<>(config);
             consumerConfig.putAll(properties);
             exitCode = maybeResetInputAndSeekToEndIntermediateTopicOffsets(consumerConfig, dryRun);
-            maybeDeleteInternalTopics(adminClient, dryRun);
+            exitCode |= maybeDeleteInternalTopics(adminClient, dryRun);
```

Comment:
should we do check both exit codes and decide whether to return 0 or 1?

Reply:
Yep. Currently it returns 1 if either exitCode is 1, and 0 otherwise. Or should we do something else?